fhd_core/visibility_manipulation/uvfits_read.pro

```diff
@@ -84,7 +84,21 @@ ENDIF ELSE BEGIN
     ;free memory
     data_array=0 
     vis_weights0=0
-    
+
+    ;; now read extensions one by one until we find the antenna table
+    t_read_ant=Systime(1)
+    ext_name = ''
+    while ext_name ne 'AIPS AN' do begin
+        ext_data = mrdfits(lun, 0, ext_header)
```

Comment:
I'm a little rusty on IDL's `mrdfits()` function, but won't this always read the first extension?

Reply:
This code wasn't changed in this branch, so I didn't test it. But from reading the `mrdfits()` documentation, I think that when the function is given an open file, it will step through the extensions one-by-one. If a nonzero number is given as the second argument, it will skip that number of extensions and read in the next one.